src/Confluent.Kafka/Admin/DeleteAclsException.cs

```diff
@@ -0,0 +1,130 @@
+// Copyright 2022 Confluent Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Refer to LICENSE for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+
+namespace Confluent.Kafka.Admin
+{
+    /// <summary>
+    ///     Represents an error that occurred during a delete ACLs request.
+    /// </summary>
+    public class DeleteAclsException : KafkaException
+    {
+        /// <summary>
+        ///     Initialize a new instance of DeleteAclsException.
+        /// </summary>
+        /// <param name="reports">
+        ///     The result corresponding to all the ACL filters in the request 
+        ///     (whether or not they were in error). At least one of these
+        ///     results will be in error.
+        /// </param>
+        public DeleteAclsException(List<DeleteAclsReport> reports)
+            : base(new Error(ErrorCode.Local_Partial,
+                $"An error occurred deleting ACLs: [{string.Join(", ", reports.Select(r => r.ToString()))}]."))
+        {
+            Reports = reports;
+        }
+
+        /// <summary>
+        ///     The result corresponding to all the delete ACLs operations in the request 
+        ///     (whether or not they were in error). At least one of these
+        ///     results will be in error.
+        /// </summary>
+        public List<DeleteAclsReport> Reports { get; }
```

Comment:
I agree that `Reports` is a better name, but other exception classes use `Results`, so i think change this.